docs/features/models/entity_linking.rst

```diff
@@ -0,0 +1,41 @@
+Entity Linking
+========================================
+
+Entity linking is the task of mapping of words in the text, corresponding to names of persons, locations and organizations, to entities in the target knowledge base (Wikidata in our case).
+
+Entity Linking component consists of the following steps:
+
+* the substring, detected with :config:`NER <ner/ner_rus_bert_probas.json>`, is fed into TfidfVectorizer with the following parameters: analyzer="char_wb", ngram_range=(2, 2), max_features=500, and the vector is converted to dense
+* `Faiss <https://github.com/facebookresearch/faiss>`__ library is used to find k nearest neighbours for tf-idf vector in the matrix where rows correspond to tf-idf vectors of words in entity titles
+* Entities are ranked by number of relations in Wikidata (number of outgoing edges of nodes in the knowledge graph)
+* :config:`BERT <classifiers/entity_ranking_bert_rus_no_mention.json>` is used for ranking of entities by description of the entity and the sentence with the mention of the entity.
```

Comment:
```suggestion
* :config:`BERT <classifiers/entity_ranking_bert_rus_no_mention.json>` is used for entities ranking by entity description and the sentence with the mention of the entity.
```
And I don't quite understand what you mean by `and the sentence with the mention of the entity`